build_scripts/appimage/AppRun

```diff
@@ -24,7 +29,12 @@ do
     fi
 done
 
+# Save HOME and USER because capsh changes them. Until ubuntu adds --noenv option we have to rely on a workaround
+export HOME_ORIGINAL=$HOME
+export USER_ORIGINAL=$USER
+
 export PYTHONNOUSERSITE=1
-(PYTHONHOME="${APPDIR}/opt/python3.11" exec "${APPDIR}/opt/python3.11/bin/python3.11" "${APPDIR}/usr/src/monkey_island.py" $@)
+
+(PYTHONHOME="${APPDIR}/opt/python3.11" exec sudo capsh --keep=1 --user=${USER_ORIGINAL} --inh=cap_net_bind_service --addamb=cap_net_bind_service -- -c "env HOME=${HOME_ORIGINAL} USER=${USER_ORIGINAL} ${APPDIR}/opt/python3.11/bin/python3.11 ${APPDIR}/usr/src/monkey_island.py" $@)
```

Comment:
I considered this approach, but is capsh guaranteed to be installed?

Reply:
Does this still work on WSL?

Reply:
When installed as a service, this will still run with `sudo capsh`. This will cause the service to hang if a password is required to run `sudo`. Furthermore, `capsh` is theoretically unnecessary since we set `AmbientCapabilities`.

The systemd unit should set an environment variable that indicates it's running as a service. The `AppRun` should only use `capsh` if the environment variable is not set.

Reply:
WSL works.
 `capsh` with `--addamb` is available in ubuntu >20 and debian >11. I haven't tested other distros, but AFAIK it should be available in the latest versions and even have `--noenv` flag that we'll want to use eventually.